src/Resources/config/app/config.yml

```diff
@@ -20,11 +20,14 @@ sylius_mailer:
             subject: Units refunded
             template: "@SyliusRefundPlugin/Email/unitsRefunded.html.twig"
 
-prooph_service_bus:
-    command_buses:
-        sylius_refund_command_bus: ~
-    event_buses:
-        sylius_refund_event_bus: ~
+framework:
+    messenger:
+        default_bus: sylius_refund_plugin.command_bus
```

Comment:
I copy-pasted it into similar PR in InvoicingPlugin, but I think we shouldn't set the default bus for the whole application here.

Reply:
Totally agreed 👍

Reply:
I've defined the default bus only [for the test env](https://github.com/Sylius/RefundPlugin/pull/89/files#diff-3cb4d392b9fb5e2d3cf5e66c4b081c7fR3). After all, we need this configuration only to be able to inject command bus to Behat contexts. This way it will be possible and also it won't be configured for the whole application 🐃